code/index.html

```diff
@@ -2,21 +2,93 @@
     <header>
       <!-- Some kind of logo and navbar -->
+      <div class="site-logo-name">
+        <img class="logo" src="./icons/icons8-sailing-ship-96.png" alt="logo" />
+        <span>SKEPP OHOJ!</span>
+      </div>
+
+      <nav>
+        <a href="#">HOME</a>
+        <a href="#">ABOUT</a>
+        <img class="logo" src="./icons/icons8-sailing-ship-96.png" alt="logo" />
+        <a href="#">CONTACT</a>
+        <a href="#">FOUR</a>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <main>
+      <section class="big-news-container">
+        <!-- The big news content -->
+        <div class="big-news-card">
+          <img src="./images/boat-1.jpg" alt="boat one" />
+        </div>
+        <div class="big-news-card big-news-card-grow">
+          <h2>Big news - Skepp Ohoj!</h2>
+          <p>
+            Lorem ipsum dolor sit amet, debitis disputando at ius, ea usu nullam
+            placerat adolescens, ex accumsan antiopam interpretaris sea. Te modo
+            inciderint sed. Has oratio suscipit id, ut malis putant copiosae
+            ius, cum mazim consul vivendum at. Eum et probo facer, mei id option
+            perfecto suscipiantur. Te alii patrioque vel, eos cu eros accusam
+            albucius. Nullam omnium petentium vel at, cum ei dico dictas
+            ocurreret.
+          </p>
+        </div>
+      </section>
+
+      <section class="other-news-container">
+        <!-- The other news  -->
+        <div class="other-news-card">
+          <img src="./images/boat-1.jpg" alt="boat one" />
+          <h3>Other news 1</h3>
+        </div>
+        <div class="other-news-card">
+          <img src="./images/boat-2.jpg" alt="boat two" />
+          <h3>Other news 2</h3>
+        </div>
+        <div class="other-news-card">
+          <img src="./images/boat-3.jpg" alt="boat three" />
+          <h3>Other news 3</h3>
+        </div>
+        <div class="other-news-card">
+          <img src="./images/boat-4.jpg" alt="boat four" />
+          <h3>Other news 4</h3>
+        </div>
+        <div class="other-news-card">
+          <img src="./images/boat-1.jpg" alt="boat one" />
+          <h3>Other news 5</h3>
+        </div>
+        <div class="other-news-card">
+          <img src="./images/boat-2.jpg" alt="boat two" />
+          <h3>Other news 6</h3>
+        </div>
+        <div class="other-news-card">
+          <img src="./images/boat-3.jpg" alt="boat three" />
+          <h3>Other news 7</h3>
+        </div>
+      </section>
+    </main>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
+    <footer>
+      <p>
+        This is made by Ylva Nilsson within the project "News site" at Technigo
+      </p>
+      <a
+        target="_blank"
+        href="https://icons8.com/icons/set/sailing-ship-small--v1"
+        >Sailing Ship icon</a
+      >icon by
+      <a target="_blank" href="https://icons8.com">Icons8</a>
```

Comment:
Be consistent (and possibly wrap the whole text or the text outside the `a` in a `p` tag):
```
<a target="_blank" href="https://icons8.com/icons/set/sailing-ship-small--v1">Sailing Ship icon</a>
<p>icon by</p>
<a target="_blank" href="https://icons8.com">Icons8</a>
```